backend/ecommerce/schema.py

```diff
@@ -143,6 +148,46 @@ def mutate(self, info, name, email, phone, password):
         )
 
 
+class LikeReview(graphene.Mutation):
+    id = graphene.String()
```

Comment:
Are you sure ? for string ?

Reply:
It works for both string and integer, but when we get the json response it appears as string so I think we should use string everywhere

Reply:
what is advised to use int or string ?

Reply:
Idt one of them is preferred as such. Just pick one and be consistent throughout the project